ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/compliance.coverage.utils.ts

```diff
@@ -23,17 +37,47 @@ export function getPassAndTotalCount(checkStats: ComplianceCheckStatusCount[]):
 }
 
 export function calculateCompliancePercentage(passCount: number, totalCount: number): number {
-    return totalCount > 0 ? (passCount / totalCount) * 100 : 0;
+    return totalCount > 0 ? Math.round((passCount / totalCount) * 100) : 0;
+}
+
+function getComplianceStatus(passPercentage: number): ComplianceStatus {
+    let status: ComplianceStatus = ComplianceStatus.SUCCESS;
+
+    if (passPercentage < DANGER_THRESHOLD) {
+        status = ComplianceStatus.DANGER;
+    } else if (passPercentage < WARNING_THRESHOLD) {
+        status = ComplianceStatus.WARNING;
+    }
+
+    return status;
 }
 
-export function getProgressBarVariant(passPercentage: number): ProgressVariant | undefined {
-    let progressVariant: ProgressVariant | undefined;
+export function getCompliancePfClassName(passPercentage: number): string {
+    const status = getComplianceStatus(passPercentage);
 
-    if (passPercentage < 50) {
-        progressVariant = ProgressVariant.danger;
-    } else if (passPercentage < 75) {
-        progressVariant = ProgressVariant.warning;
+    if (status === ComplianceStatus.DANGER) {
+        return 'pf-m-danger';
+    }
+    if (status === ComplianceStatus.WARNING) {
+        return 'pf-m-warning';
     }
+    return '';
+}
 
-    return progressVariant;
+export function getComplianceLabelGroupColor(
+    passPercentage: number | undefined
+): LabelColor | undefined {
+    if (passPercentage === undefined) {
+        return undefined;
+    }
+
+    const status = getComplianceStatus(passPercentage);
+
+    if (status === ComplianceStatus.DANGER) {
+        return 'red';
+    }
+    if (status === ComplianceStatus.WARNING) {
+        return 'gold';
+    }
+    return 'blue';
```

Comment:
On Monday, let’s double check the PatternFly colors to see whether these colors are consistent with danger and warning above. If I remember correctly, we found and removed a subtle mismatch in Vulnerabilities.

Reply:
I found the coloring a little odd here too but both background colors appear to align with the respective status

- danger/red -> #faeae8
- warning/gold -> #fdf7e7